### DISCUSSION TITLE
FabricMC/javapoet not in Maven central repository

### DISCUSSION THREAD

**Discussion by pawellabaj** [FabricMC/fabric-api#3032] [FAQ] [answered]
I'd like to ask if is there any particular reason that FabricMC/javapoet is not available in Maven central repository.

**Comment by warjort** [comment, marked as answer]
Probably because it is a fork used to apply a patch that was not accepted in the main repo?

https://github.com/square/javapoet/discussions/866#discussioncomment-2138815
https://github.com/square/javapoet/pull/840#issuecomment-1033191908

**Reply by pawellabaj** [reply]
The situation is really strange and confusing
